repo/mock.go

```diff
@@ -28,6 +28,10 @@ func (m *Mock) SetConfig(updated *config.Config) error {
 	return nil
 }
 
+func (m *Mock) BackupConfig(prefix string) (string, error) {
+	return "config-" + prefix + "-backup", nil
```

Comment:
maybe this can just return `errTODO` like the other methods? If we arent actually going to back up the config, we probably should bother to pretend with the name

Reply:
done